.vscode/launch.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
If were adding a .vscode, I think we'll want to make these slightly more unique to our project, perhaps in another commit?